app/dashboard/models.py

```diff
@@ -2028,6 +2033,23 @@ def point_value(self):
     def i18n_name(self):
         return _(next((x[1] for x in self.ACTIVITY_TYPES if x[0] == self.activity_type), 'Unknown type'))
 
+    @property
+    def text(self):
+        from django.template.loader import render_to_string
+        from bs4 import BeautifulSoup
```

Comment:
We should move these imports to module scope

Reply:
done